spec/dummy/app/models/region.rb

```diff
@@ -1,3 +1,12 @@
 class Region < ActiveRecord::Base
   belongs_to :country
+  has_many :cities, dependent: :destroy
+
+  def self.ransackable_attributes(auth_object = nil)
+    ["country_id", "created_at", "id", "information", "name", "updated_at"]
+  end
+
+  def self.ransackable_associations(auth_object = nil)
```

Comment:
⚠️ **[rubocop]** <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>[Correctable] Lint/UnusedMethodArgument: Unused method argument - auth_object. If it's necessary, use _ or _auth_object as an argument name to indicate that it won't be used. If it's unnecessary, remove it. You can also write as ransackable_associations(*) if you want the method to accept any arguments but don't care about them.